yb-voyager/src/query/queryissue/constants.go

```diff
@@ -432,13 +432,35 @@ const (
 To address this issue and improve query performance, application-level sharding is recommended. Refer to the docs for more details. Ensure that the index on the column is configured to be range-sharded.
 Note: If the table is created as colocated, this hotspot concern can safely be ignored, as all the data resides on a single tablet, and the distribution is no longer relevant.`
 
-	REDUNDANT_INDEXES="REDUNDANT_INDEXES"
-	REDUNDANT_INDEXES_ISSUE_NAME="Redundant index"
-	REDUNDANT_INDEXES_DESCRIPTION="Redundant indexes can be dropped when a stronger index is present, meaning an index that includes the same initial key columns (in order) and may extend with additional keys, thus fully covering the redundant one."
+	REDUNDANT_INDEXES             = "REDUNDANT_INDEXES"
+	REDUNDANT_INDEXES_ISSUE_NAME  = "Redundant index"
+	REDUNDANT_INDEXES_DESCRIPTION = "Redundant indexes can be dropped when a stronger index is present, meaning an index that includes the same initial key columns (in order) and may extend with additional keys, thus fully covering the redundant one."
```

Comment:
nit: 
"when a stronger index is present, meaning an index that" -> 
"when a stronger index is present. A stronger index is one that"